Project2015To2017.Migrate2017.Tool/Project2015To2017.Migrate2017.Tool.csproj

```diff
@@ -9,10 +9,6 @@
 		<PackageId>Project2015To2017.Migrate2017.Tool</PackageId>
 		<Product>Project2015To2017.Migrate2017.Tool</Product>
 		<OutputType>Exe</OutputType>
-		<RestoreAdditionalProjectSources>
```

Comment:
I removed NuGet dependency because that imposed security risks. Different code was used in published NuGet packages and during development - that's horrible. But now we must keep our vendored source code relatively up-to-date. Some tradeoffs are present.